### PR TITLE
config updated

### PR DESCRIPTION
**Pull request by JosepSampe**
Updated config_template.yaml #383 
Updated docs
[ce] Set `remote_invoker=True` in code_engine config file, this way users don't need to set it manually in config file
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by gilv** [comment]
@JosepSampe can we merge it?

**Comment by JosepSampe** [comment]
@gilv If you agree with the changes, yes

File: config/config_template.yaml [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
     #access_key : <ACCESS_KEY>  # Optional
     #secret_key : <SECRET_KEY>  # Optional
 
+#ibm_vpc:
+    #endpoint    : <REGION_ENDPOINT>
+    #instance_id : <INSTANCE_ID>
+    #ip_address  : <FLOATING_IP_ADDRESS>
+
 #rabbitmq:
     #amqp_url   : <RABBIT_AMQP_URL>  # amqp://
```
Comment (gilv) [inline config/config_template.yaml:50]: @JosepSampe and where is section how to use ssh with just regular vm?

Comment (gilv) [reply config/config_template.yaml:50]: @JosepSampe i think you just need to copy-paste the vm part from the PR itself

Comment (JosepSampe) [reply config/config_template.yaml:55]: @gilv added

**Pull request merged**